uat/testing-features/src/main/java/com/aws/greengrass/platforms/linux/NetworkUtilsLinux.java

```diff
@@ -0,0 +1,43 @@
+
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.platforms.linux;
+
+import com.aws.greengrass.platforms.NetworkUtils;
+import lombok.AllArgsConstructor;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+@AllArgsConstructor
+public class NetworkUtilsLinux extends NetworkUtils {
+    private static final String DISABLE_OPTION = "--delete";
+    private static final String APPEND_OPTION = "-A";
+    private static final String IPTABLES_DROP_DPORT_EXTERNAL_ONLY_COMMAND_STR
+            = "sudo iptables %s INPUT -p tcp -s localhost --dport %s -j ACCEPT && "
+            + "sudo iptables %s INPUT -p tcp --dport %s -j DROP && "
+            + "sudo iptables %s OUTPUT -p tcp -d localhost --dport %s -j ACCEPT && "
+            + "sudo iptables %s OUTPUT -p tcp --dport %s -j DROP";
+
+    @Override
+    public void disconnectMqtt() throws InterruptedException, IOException {
+        modifyMqttConnection(APPEND_OPTION);
+    }
+
+    @Override
+    public void recoverMqtt() throws InterruptedException, IOException {
+        modifyMqttConnection(DISABLE_OPTION);
+    }
+
+    private void modifyMqttConnection(String action) throws IOException, InterruptedException {
+        for (String port : MQTT_PORTS) {
+            new ProcessBuilder().command(
```

Comment:
OTF `pillbox` should be used to run commands so that these commands can run remotely instead of only on the device which is executing OTF

Reply:
Correct, they need to run where nucleus runs. Not where the test is running

Reply:
From my point of view due to missing control of maven repo when OTF is published.
I can do that by publishing to local maven but that can poison the maven cache

Reply:
I don't understand.

Make a pul request with the right change in OTF and it can be merged and released as part of 1.2.0-SNAPSHOT.

To test, yes you locally build and install OTF. That is normal, how else would you do it.

Reply:
Platform and LogSteps code has been moved to OTF in that PR https://github.com/aws-greengrass/aws-greengrass-testing/pull/211